corehq/apps/data_cleaning/templatetags/data_cleaning_filters.py

```diff
@@ -0,0 +1,28 @@
+from django import template
+from django.template.loader import render_to_string
+from django.utils.safestring import mark_safe
+
+from corehq.apps.data_cleaning.models import DataType
+
+register = template.Library()
+
+
+@register.filter
+def dc_filter_value(dc_filter):
+    """
+    Renders an appropriately formatted value for a
+    `BulkEditFilter` based on the `data_type`.
+    """
+    if dc_filter.value is None:
+        return ""
+    context = {}
+    if dc_filter.data_type == DataType.MULTIPLE_OPTION:
+        context['values_list'] = dc_filter.value.split(" ")
+    else:
+        context['value'] = dc_filter.value
+    return mark_safe(  # nosec: render_to_string should have already handled escaping
```

Comment:
Is this referring to the next line, or to a literal earlier `render_to_string` call? I can't find an earlier call. I see `list_item.html` is only used in `ManageFiltersFormView`, which is presumably an HTMX view since it's only part of a page, so maybe `render_to_string` gets called implicitly by the HTMX mixin? I don't quite remember how this all fits together.

Having a `mark_safe` inside a template tag, which could be reused elsewhere, ought to be very clear. I might rename this to `dc_filter_safe_value`.

Reply:
This is the one question I want to understand better before approving.

Reply:
`render_to_string` is nested inside. it's right below this line...see line 24.

we use this same `nosec` notice in another place where we follow the same pattern in a templatetag:

https://github.com/dimagi/commcare-hq/blob/f861e4748fc08f306278074bc95fe385e937e14c/corehq/apps/hqwebapp/templatetags/hq_shared_tags.py#L141-L146